arangod/RestHandler/RestAdminClusterHandler.cpp

```diff
@@ -2070,6 +2075,157 @@ async<void> RestAdminClusterHandler::handleNumberOfServers() {
   }
 }
 
+async<void> RestAdminClusterHandler::handleUniqId() {
+  if (!ServerState::instance()->isCoordinator()) {
+    generateError(rest::ResponseCode::FORBIDDEN, TRI_ERROR_HTTP_FORBIDDEN,
+                  "only allowed on coordinators");
+    co_return;
+  }
+
+  // Only PUT method is allowed and always requires admin privileges
+  if (!ExecContext::current().isAdminUser()) {
+    generateError(rest::ResponseCode::FORBIDDEN, TRI_ERROR_HTTP_FORBIDDEN);
+    co_return;
+  }
+
+  switch (request()->requestType()) {
+    case rest::RequestType::PUT:
+      co_return co_await handlePutUniqId();
+    default:
+      generateError(rest::ResponseCode::METHOD_NOT_ALLOWED,
+                    TRI_ERROR_HTTP_METHOD_NOT_ALLOWED);
+      co_return;
+  }
+}
+
```

Comment:
Could you briefly summarize here what this RestHandler call is supposed to do?
Especially what the two modes are and their use-case

Reply:
Also what the side-effect impact is

Reply:
I have added an explanation before the implementation.